crates/sui-protocol-config/src/lib.rs

```diff
@@ -634,6 +643,7 @@ impl ProtocolConfig {
                 reward_slashing_rate: Some(5000),
                 storage_gas_price: Some(1),
                 max_transactions_per_checkpoint: Some(1000),
+                max_checkpoint_size: Some(30 * 1024 * 1024),
```

Comment:
So we landed on 30MiB for the size of a checkpoint?

Reply:
My thinking was this would leave enough overhead for me to set soft msg size limit on state sync to 32MB